docs/guide/collections/configuration-reference.md

```diff
@@ -44,6 +44,14 @@ The string that is used when a UI needs a title for the collection. Defaults to
 title: 'My Favourite Website Layouts'
 ```
 
+#### root <Badge text="added in v1.5.0" type="tip"/>
+
+Pull the collection to the navigation top level, next to the defaults "Components" and "Documentation".
```

Comment:
maybe worth mentioning that this only works for component collections, I believe I did not add support for doc collections nor did I test this.

Reply:
✅